solvers/QuadraticProgram.m

```diff
@@ -52,8 +52,7 @@
     obj = obj.addBoundingBoxConstraint(BoundingBoxConstraint(x_lb,x_ub));
     obj.Q = Q;
     obj.f = f;
-  % todo: check dependencies here and pick my favorite that is also installed
-    obj.solver = 'gurobi';
+    obj = obj.setSolver('default');
```

Comment:
this call is in the constructor, which triggers the logic in setSolver.  when you call setSolver('default') it checks dependencies, and picks our favorite 
On Apr 12, 2015, at 11:08 AM, Michael Posa <notifications@github.com> wrote:

> In solvers/QuadraticProgram.m:
> 
> > @@ -52,8 +52,7 @@
> >      obj = obj.addBoundingBoxConstraint(BoundingBoxConstraint(x_lb,x_ub));
> >      obj.Q = Q;
> >      obj.f = f;
> > -  % todo: check dependencies here and pick my favorite that is also installed
> > -    obj.solver = 'gurobi';
> > +    obj = obj.setSolver('default');
> Do you need to set the solver to default? Since it's the default.
> 
> —
> Reply to this email directly or view it on GitHub.
>